package.json

```diff
@@ -34,7 +34,6 @@
     "angular": "^1.5.3",
     "ng-metadata": "^1.5.2",
     "reflect-metadata": "^0.1.3",
-    "system-text": "^0.1.0",
```

Comment:
Hi @Oipo have you tried to remove `systemjs-plugin-text` and instead keep and use `system-text`
Becouse the official npm released plugin is `system-text` the other one is only on github and was never published. And more than this, using `system-text` may avoid the `traceour`dependency because I've just used it successfully in my branches without any need to include traceour. Can you give it a try?